Create Blog Post “re-building-a-“dead”-tech-community” (#2120)
Nexmo/deved-platform: content/blog/en/re-building-a-“dead”-tech-community.md

---
title: Re-Building a “Dead” Tech Community
description: Tips and lessons about rebuilding a dormant tech community. In this
  example, a Ruby on Rails meetup group; Developer Relations.
thumbnail: /content/blog/re-building-a-“dead”-tech-community/rebuilding-community-1.png
author: benjamin-aronov
published: true
published_at: 2022-05-25T02:30:10.175Z
updated_at: 2022-05-25T02:30:10.207Z
category: devlife
tags:
  - ruby
  - community
  - devrel
comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---

> In 2018 I wrote a [post](https://medium.com/le-wagon/yes-we-still-teach-ruby-19d3d9b163e) about learning and teaching Ruby long after the (local) tech world considered it a dead language. For all my love for Ruby and Rails, one thing that was sorely missing was an active, local community in Tel Aviv. 

Reviewer: ```suggestion
In 2018 [I wrote a blog post](https://medium.com/le-wagon/yes-we-still-teach-ruby-19d3d9b163e) about learning and teaching Ruby long after the (local) tech world considered it a dead language. For all my love for Ruby and Rails, one thing that was sorely missing was an active, local community in Tel Aviv. 
```

Reviewer: longer link for SEO

Fast forward to 2022, and a few weeks ago I attended the first in-person [Israel.rb](https://www.facebook.com/groups/272757750683415) meetup. There were 2 amazing talks, food and drinks at a gorgeous office, Ruby job openings, and 30+ very much alive Rubyists. 

> So what happened in between? I helped build the community (with two amazing co-organizers!) that I so desperately wanted. So let’s celebrate this milestone with some thoughts, observations, and hopefully lessons for someone somewhere.

Reviewer: ```suggestion
So what happened in-between? I helped build the community (with two amazing co-organizers!) that I so desperately wanted. So let’s celebrate this milestone with some thoughts, observations, and hopefully lessons for someone, somewhere.
```

> ![](https://cdn-images-1.medium.com/max/1200/1*yDO_w6fZlY4TCd_Cv5BTrg.png)

Reviewer: alt. text for visibility please

## Background

#### The Problem

> Back in 2018, I finished the Le Wagon Bootcamp. I had signed up only a couple of weeks before moving to Tel Aviv, knowing that my basic level of programming was not enough for the competitive Israeli industry. I had a feeling, *“it’s not what you know, it’s who you know”.* I thought Le Wagon would give me that network.

Reviewer: Can we have a link to the bootcamp, just so people know what it is?

A few months later I was ecstatic when Ruby Underground announced a meetup. I made sure to clear my calendar and attend. The talks were super interesting, the two alumni I dragged along met with WeWork HR looking to hire, and I even got a few minutes to talk to the meetup organizer. It all seemed to be falling into place. That is until I followed up with the organizer. Again, radio silence. In hindsight, this was 2019 at WeWork as the company was in a period of flux with an upcoming and then canceled IPO. 

> *It became clear that Ruby was no longer a priority for even top Ruby organizers in Tel Aviv.*

## A  Meetup Is Formed

#### The Co-Stars

By the fall of 2019 something amazing happened that changed my life. I attended a meetup.

To be specific, I attended the first DEV IRL meetup in Tel Aviv. It was organized by Ben Greenberg and Avital Tzubeli. Avital had spoken at Le Wagon and we were quickly becoming friends. Ben was one of the few locals talking Ruby on the internet, so obviously I was a big fanboy. My coworker at the time Clara Morgen had been accepted to give a talk and I rallied the troops to support our lead teacher.

> After spending all of 5 minutes on Vonage’s rooftop. I was sold! I was the one now getting in contact with HR. Within 6 months both I and Avital would join Ben at Vonage. And Israel.rb would be set in motion.

Reviewer: ```suggestion
After spending all of 5 minutes on Vonage’s rooftop, I was sold! I was the one now getting in contact with HR. Within 6 months both Avital and I would join Ben at Vonage, and Israel.rb would be set in motion.
```

*The Four Speakers at DEV IRL: [Jonathan](https://twitter.com/johnnymakestuff), [Ben](https://twitter.com/rabbigreenberg?lang=en), [Avital](https://twitter.com/avitaltzubeli), and [Clara](https://twitter.com/Clara_Morgen)*

> ![](https://cdn-images-1.medium.com/max/1200/0*tazK_h18hRXy88x0.jpg)

Reviewer: alt text for accessibility please

> I started my new position at Vonage in February 2020. Yes. That February 2020. The one with the beginning of a global pandemic. Great timing!

Reviewer: ```suggestion
I started my new position at Vonage in February 2020. Yes, _that_ February 2020. The one with the beginning of a global pandemic. Great timing!
```

> Even as we entered uncertain times, my boss at the time Olia continued to encourage me to find ways to make work meaningful and meet personal goals. One big obvious way: exploiting my new access to one of the coolest Ruby devs in Israel, and now colleague Mr. Greenberg.

Reviewer: ```suggestion
Even as we entered uncertain times, my boss at the time Olia continued to encourage me to find ways to make work meaningful and meet personal goals. There was one big, obvious opportunity here: exploiting my new access to one of the coolest Ruby devs in Israel (and now colleague) Mr. Greenberg.
```

It didn’t take much convincing. Ben was in!

> Ben had a great insight; “let’s bring in a third organizer so the group wasn’t 100% Vonage”. And so Ariel Caplan, Ruby Backend extraordinaire completed our trifecta. At the time Ariel was at Cloudinary and had already spoken at both RubyConf and RailsConf multiple times. Oh and that one last Ruby Underground I attended? Ariel was a speaker so I knew he was equally brilliant and intelligible for all levels.

Reviewer: ```suggestion
Ben had a great insight; “let’s bring in a third organizer so the group wasn’t 100% Vonage”. And so Ariel Caplan, Ruby Backend extraordinaire completed our trifecta. At the time Ariel was at Cloudinary and had already spoken at both RubyConf and RailsConf multiple times. Oh, and that one last Ruby Underground I attended? Ariel was a speaker so I knew he was equally brilliant and intelligible for all levels.
```

*Ariel, Ruby speaker extraordinaire*![](https://cdn-images-1.medium.com/max/1200/0*J4hw02z7vn69vxtW.jpg)

So the A-Team was:

Ben Greenberg: DevRel, API Builder, Community Organizer, Twitter Superstar

Ariel Caplan: Master Programmer, Living Proof That Ruby Can Scale

Me: Makes Ok-ish Graphics, Social Media Spammer, Knows Ruby Juniors

#### **The Metrics**

> Ok, so that’s how we got started. And what’s actually happened? 

Reviewer: ```suggestion
Ok, so that’s how we got started. So, what’s actually happened? 
```

**3 Online Meetups:** 

* July 15, 2020

  * 30 attendees
  * Speakers: Ben Greenberg, Allison McMillian (Github)
  * Sponsors: Vonage, Cloudinary
* August 12, 2020

  * 25 attendees
  * Speakers: Jonan Scheffler (New Relic), Ariel Caplan
  * Sponsors: Vonage, Cloudinary
* October 12, 2020

  * 12 attendees
  * Speakers: Dan Moore (FusionAuth)
  * Sponsors: Vonage, Cloudinary

*Allison McMillian at our very first meetup!*![](https://cdn-images-1.medium.com/max/1200/0*V3HCpTsFoD1LVbHt)

**1 Hybrid Mini-Conference:**

* [EMEA on Rails](https://www.emeaonrails.com/)
* June 9, 2021

  * 30 in-person attendees, 120 online attendees
  * 12 speakers

>   * Sponsors: Jet Brains, Signal Wire, Le Wagon, Vonage, Cloudinary, Orbit, Zerus & Ona

Reviewer: ```suggestion
  * Sponsors: JetBrains, Signal Wire, Le Wagon, Vonage, Cloudinary, Orbit, Zerus & Ona
```

**1 In-Person Meetup:**

* April 11, 2022

  * 30 attendees
  * Speakers: Yoel Blum (Tailor Brands), Maya Shavin (Microsoft)
  * Sponsor: Tailor Brands

## Lessons and the Future

**Start Now!**

> Like studying or exercising, the biggest hurdle of building a community is getting started. You might think of all the reasons to put off building a community. But at the end of the day, you are definitely going to make mistakes. The sooner you get started, the quicker you can make those mistakes and iterate your way to success.

Reviewer: ```suggestion
Like studying or exercising, the biggest hurdle of building a community is getting started. You might think of all the reasons to put off building a community but at the end of the day, you are definitely going to make mistakes. The sooner you get started, the quicker you can make those mistakes and iterate your way to success.
```

As you can see, I had a few advantages to get off the ground. But the basics of any (dead tech) community are always the same:

**PEOPLE + CONTENT + PLACE**

The great thing about trying to revive a dead tech community is that there is probably already lots of groundwork laid for you. In each of the aspects of a community (P+C+P), you’re just a few google searches away from finding troves of help.

> Finding people interested in your specific language or framework, that are also in your location is probably the hardest part of reviving a community. The good thing is you probably already know a few. I highly recommend asking one or two of them to join you in forming the new community, even if the work is divided unequally. Each person brings their own network and the difference between starting with a potential network of 5 versus 10 or 15 can be critical early on.

Reviewer: ```suggestion
Finding people interested in your specific language or framework that are also in your location is probably the hardest part of reviving a community. The good thing is you probably already know a few. I highly recommend asking one or two of them to join you in forming the new community, even if the work is divided unequally. Each person brings their own network and the difference between starting with a potential network of 5 versus 10 or 15 can be critical early on.
```

One more secret to finding great content in a post-2020 world is that remote life is no longer stigmatized. Israel.rb was able to really take advantage of this, bringing in world-class Rubyists that would have never been able to make it to Tel Aviv from around the world for a meetup. Now with in-person events opening up, you can get the best of both worlds. Meet with your local group in person and bring in a top expert for a live watch party!

Lastly, you’ll need a place. That means a place for meeting. Is it in person? Try coworking offices, incubators, and programming bootcamps. They all usually have everything needed for meetups and are happy to host for free. Even better is to find a company working in your tech stack and let them use some HR budget on you. It’s best to set expectations ahead of time about turnout. Many eager HR reps might not realize that dying communities may mean 10–20 attendees is a good result. If they are expecting 20–50+, there may be ill will and you can lose one of your best potential partners.

**Going Hybrid**

> Or you can also opt for virtual/hybrid meetups. We learned a very big lesson with our hybrid event. It turned out that while the local community of Rubyists was small, there are still many loyal Rubyists in our region. We thought we’d be able to split in-person and online duties between the organizers. What we found out was that it’s really like running two seperate events. You need one set of organizers/moderators for the online and one for the in-person.

Reviewer: ```suggestion
You can also opt for virtual/hybrid meetups; we learned a very big lesson with our hybrid event. It turned out that while the local community of Rubyists was small, there are still many loyal Rubyists in our region. We thought we’d be able to split in-person and online duties between the organizers. What we found out was that it’s really like running two separate events. You need one set of organizers/moderators for the online and one for the in-person.
```

**Consistency**

> One area that we struggled with was consistency. We had our second meetup one month after our first. Then the third was two months later. Then the fourth one was 8 months later! I know I said the biggest and best thing is to just start and get going. But if you can, have a plan when you launch. How will the first event link to future content? Maybe you won’t have monthly meetups. Maybe it will only be quarterly, that’s ok. But how will you keep the energy going in the meantime? Will you be posting content online in between? Will you be doing outreach for future speakers at your first event? Can you have easy, ready-made objectives and ways for attendees to get involved from the get-go? We tried some of these but we didn’t really have a plan and we were really inconsistent in the way we organized meetups. As you can see, at the beginning there is always a lot of excitement for something new. How will you channel that into longevity?

Reviewer: ```suggestion
One area that we struggled with was consistency. We had our second meetup one month after our first. Then the third was two months later. Then the fourth one was 8 months later! I know I said the biggest and best thing is to just start and get going. But if you can, have a plan when you launch. How will the first event link to future content? Maybe you won’t have monthly meetups. Maybe it will only be quarterly, that’s OK. But how will you keep the energy going in the meantime? Will you be posting content online in between? Will you be doing outreach for future speakers at your first event? Can you have easy, ready-made objectives and ways for attendees to get involved from the get-go? We tried some of these but we didn’t really have a plan and we were really inconsistent in the way we organized meetups. As you can see, at the beginning there is always a lot of excitement for something new. How will you channel that into longevity?
```

**Tools**

> Along with consistent planning. Agree with your co-organizers from the beginning on consistent tooling. Each organizer in our A-team had a different preferred method for communication. We also had different preferred methods for engaging the community. And it caused a lot of problems. At first, we opened a Slack workspace, thinking this would be the home of our new community. But quickly realized that it didn’t fit our community. Slack is too cumbersome and if our maximum potential community is in the dozens, how many will actually remember to open it often enough? In the end, it became our place to organize meetups, with more friction than function.

Reviewer: ```suggestion
Along with consistent planning, you'll need to agree with your co-organizers from the beginning on consistent tooling. Each organizer in our A-team had a different preferred method for communication. We also had different preferred methods for engaging the community, and it caused a lot of problems. 

At first, we opened a Slack workspace, thinking this would be the home of our new community. But quickly realized that it didn’t fit our community. Slack is too cumbersome and if our maximum potential community is in the dozens, how many will actually remember to open it often enough? In the end, it became our place to organize meetups, with more friction than function.
```

You need to think about your local community and find the best fit for each function. How will you communicate with community members? Twitter? Facebook? Email? Telegram? Will you create a specific community hub like Discord or a web forum or a Facebook Group? How will you handle event registrations? Meetup or Eventbrite or Luma?

I highly recommend checking out the [Orbit community tools index](https://orbit.love/community-tools-index).

- - -

So you’ve wrangled a friend or two into helping you revive this tech community, together y’all have made some lists of potential community members, and some potential speakers, and you’ve even lined up a venue with a date and time. So what’s next? Launch!

> ![](https://cdn-images-1.medium.com/max/1200/0*una1z57jSuNgtw5n)

Reviewer: alt text for accessibility please

2. Understand your main stakeholders. Since launching Israel.rb, the biggest new, unexpected community member type are small to medium companies that are looking for new employees. We created a simple [Job Board](https://airtable.com/shr2EZCTlzE73skSH/tblxs9zBQANucf8y2) but are thinking about how to better incorporate them into our community. 
3. Create a landing page. As we found, jumping from tool to tool, the “frontpage” of our community was inconsistent, and hard to find valuable information. Having a single source of truth landing page that then points to all your different initiatives will help attract new members. You can use the Israel.rb \[landing page mockup](<1. <https://www.figma.com/proto/w8dEFZC7dieuFb8OL4qD6p/Untitled?page-id=0%3A1&node-id=3%3A2&viewport=619%2C233%2C0.25&scaling=scale-down>>) for inspiration.

   Have some clear community initiatives that they can find on the landing page, these are ours:

>    * **meetups:** pictures of former meetups will provide proof this an active community. Adding videos of past talks will provide instant value and show the quality of the group.

Reviewer: ```suggestion
   * **meetups:** pictures of former meetups will provide proof that this is an active community. Adding videos of past talks will provide instant value and show the quality of the group.
```

   * **discussions:** link to an online space for discussions and sharing news between events. We use a Facebook group as Facebook continues to be popular in Israel.

>    * j**ob board**: this may be the single most valuable initiative you engage in, helping folks in their career.

Reviewer: ```suggestion
   **job board**: this may be the single most valuable initiative you engage in, helping folks in their career.
```

Reviewer: I'd also capitalise all of these bullets personally

   * **state of your tech in your location:** create a list of companies that are using your technology in your area. Identifiable logos will provide a lot of social proof that this is still a cool area to be in.

![Ben Greenberg being warm and welcoming at the last meetup.](/content/blog/re-building-a-“dead”-tech-community/ben-greenberg-tailorbrands-1.png "Ben Greenberg being warm and welcoming at the last meetup.")

### Do it!

If you’re thinking about meeting with others in your field, there are probably others thinking the same. The feeling of bringing together the no-longer cool kids in developer land is honestly incredible. Stop thinking and just do it! All those companies running legacy code are relying on you!

> If you have questions about tech communities, the Vonage developer relations team is full of very talented tech community builders. [Join us](https://developer.vonage.com/community/slack) on Developer Community Slack!

Reviewer: ```suggestion
If you have questions about tech communities, the Vonage Developer Relations team is full of very talented tech community builders. [Join us](https://developer.vonage.com/community/slack) on Developer Community Slack!
```

> If you are interested in building developer communities professionally, the Vonage developer relations team is growing! Check out our [job openings](vonage.dev/join).

Reviewer: ```suggestion
If you are interested in building developer communities professionally, the Vonage Developer Relations team is growing! Check out our [job openings](vonage.dev/join).
```
